packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -253,87 +420,104 @@ import {repeat} from 'lit/directives/repeat.js';
-moving DOM when required, and is generally the most efficient way to use
-`repeat` since it performs minimum unnecessary work for insertions and removals.
+When `condition` is true, returns the result of calling `trueCase()`, else returns the result of calling `falseCase()` if `falseCase` is defined.
 
-{% switchable-sample %}
+This is a convenience wrapper around a ternary expression that makes it a
+little nicer to write an inline conditional without an else.
 
 ```ts
-@customElement('my-element')
 class MyElement extends LitElement {
-
-  @property()
-  items: Array<{id: number, name: string}> = [];
-
   render() {
     return html`
-      <ul>
-        ${repeat(this.items, (item) => item.id, (item, index) => html`
-          <li>${index}: ${item.name}</li>`)}
-      </ul>
+      ${when(this.user, () => html`User: ${this.user.username}`, () => html`Sign In...`)}
     `;
   }
 }
 ```
 
+### choose
+
+<table>
+<thead><tr><th></th><th></th></tr></thead>
+<tbody>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Import</td>
+<td class="wide-cell">
+
 ```js
-class MyElement extends LitElement {
-  static properties = {
-    items: {},
-  };
+import {choose} from 'lit/directives/choose.js';
+```
 
-  constructor() {
-    super();
-    this.items = [];
-  }
+</td>
+</tr>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Signature</td>
+<td class="wide-cell">
 
-  render() {
-    return html`
-      <ul>
-        ${repeat(this.items, (item) => item.id, (item, index) => html`
-          <li>${index}: ${item.name}</li>`)}
-      </ul>
-    `;
-  }
-}
-customElements.define('my-element', MyElement);
+```ts
+choose<T, V>(
+  value: T,
+  cases: Array<[T, () => V]>,
+  defaultCase?: () => V
+)
 ```
+</td>
+</tr>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Usable location</td>
+<td class="wide-cell">
 
-{% endswitchable-sample %}
+Any
 
-If no `keyFn` is provided, `repeat` will perform similar to a simple map of
-items to values, and DOM will be reused against potentially different items.
+</td>
+</tr>
+</tbody>
+</table>
 
-See [When to use map or repeat](/docs/templates/lists/#when-to-use-map-or-repeat) for a discussion
-of when to use `repeat` and when to use standard JavaScript flow control.
+Chooses and evaluates a template function from a list of cases based on matching
+the given `value` to a case.
 
-Explore `repeat` more in the [playground](/playground/#sample=examples/directive-repeat).
+Cases are structured as `[caseValue, func]`. `value` is matched to
+`caseValue` by strict equality. The first match is selected. Case values
+can be of any type including primitives, objects, and symbols.
 
-## templateContent
+This is similar to a switch statement, but as an expression and without
+fallthrough.
 
-Renders the content of a `<template>` element.
+```ts
+class MyElement extends LitElement {
+  render() {
+    return html`
+      ${choose(this.section, [
+        ['home', () => <h1>Home</h1>]
+        ['about', () => <h1>About</h1>]
+      ],
```

Comment:
The two template literals above are missing the `html` tag. Maybe playing with JSX too much recently? 😉

Reply:
Oops!